packages/expo-modules-autolinking/src/platforms/android.ts

```diff
@@ -135,9 +161,24 @@ async function findAndroidPackagesAsync(modules: ModuleDescriptor[]): Promise<st
 }
 
 /**
- * Converts the package name to Android's project name.
- * Example: `@unimodules/core` → `unimodules-core`
+ * Converts the package name and gradle file path to Android's project name.
+ *   `$` to indicate subprojects
+ *   `/` path will transform as `-`
+ *
+ * Example: `@unimodules/core` + `android/build.gradle` → `unimodules-core`
```

Comment:
Let's stop referring to unimodules packages.
```suggestion
 * Example: `@expo/example` + `android/build.gradle` → `expo-example`
```